src/dispatch/incident/service.py

```diff
@@ -35,23 +35,33 @@ def resolve_incident_commander_email(
     page_commander: bool,
 ):
     """Resolves the correct incident commander email based on given parameters."""
-    commander_service = incident_type_service.get_by_name(
-        db_session=db_session, name=incident_type
-    ).commander_service
-
-    p = plugin_service.get_active(db_session=db_session, plugin_type="oncall")
-
-    # page for high priority incidents
-    # we could do this at the end but it seems pretty important...
-    if page_commander:
-        p.instance.page(
-            service_id=commander_service.external_id,
-            incident_name=incident_name,
-            incident_title=incident_title,
-            incident_description=incident_description,
-        )
+    # We resolve the incident commander email
+    # default to reporter if we don't have an oncall plugin enabled
+    oncall_plugin = plugin_service.get_active(db_session=db_session, plugin_type="oncall")
+    if oncall_plugin:
```

Comment:
I think it will be easier to read if you check if we don't have the `oncall_plugin`
```suggestion
    if not oncall_plugin:
        return reporter_email
```